signer/core/api.go

```diff
@@ -105,6 +110,30 @@ type Metadata struct {
 	Origin    string `json:"Origin"`
 }
 
+type SigFormat struct {
+	Mime		string
+	ByteVersion	byte
+}
+
+var (
+	ApplicationValidator = SigFormat{
+		"application/validator",
+		0x00,
+	}
+	ApplicationClique = SigFormat{
+		"application/clique",
+		0x01,
+	}
+	DataPlain = SigFormat{
+		"data/plain",
```

Comment:
Should not the `0x45` version, which is 'Ethereum Signed Message' be `text/plain'? Because it's definitely not good for binary data

Reply:
Hmm I see so are you saying it can be confusing? If so, is it okay to set `data/structured`, which is EIP-712 essentialy, as `0x45`? I'd keep it alphabetically ordered, although I'm not sure if it'd be better to just keep at defined in the EIP-191 specs (that is, `0x45` represents plain messages).

Reply:
As of the latest implementation, the byte version fully respects EIP-191.